src/virtio_console.js

```diff
@@ -0,0 +1,294 @@
+
+"use strict";
```

Comment:
If you have used some document/reference for the protocol, you should post it here in a comment.

Reply:
https://docs.oasis-open.org/virtio/virtio/v1.2/csd01/virtio-v1.2-csd01.html#x1-2900003